ner_processing/data.py

```diff
@@ -0,0 +1,111 @@
+from typing import List, Any
+from PyQt6.QtCore import QDateTime
+
+
+class Data:
+    """
+    Wrapper class for an individual piece of data.
+    """
+
+    def __init__(self, timestamp: QDateTime, id: int, value: Any):
+        self.timestamp = timestamp
+        self.id = id
+        self.value = value
+
+    def __str__(self):
+        """
+        Overrides the string representation of the class.
+        """
+        return f"ID {self.id} - {self.timestamp} - {self.value}"
+
+    
+class ProcessData:
+    """
+    Utility functions to process message data.
+    """
+
+    @staticmethod
```

Comment:
What is the purpose of @staticmethod here? Is it just good formatting since the methods aren't specific to a specific object's instance?